Updating russian localization for 1.19.2

Edarlingen opened the pull request:
- Added missing translation lines for items "Manual Compressor" and "Solar Compressor"
- Synced variable names `pneumaticcraft.gui.tab.info.item.armor.*` with en_us locale
- Fixed various typos and grammatical errors

Commit:
Updating russian localization

desht commented:
I can see a lot of minor spelling changes to words (adding/removing one character), in words which still translate into the same English word (via google translate).

What's the nature of these changes? Is there some difference of dialect here?

Edarlingen commented:
That's because this words have typos in them. All type of automatic translators could possibly fix this typos on the fly even if words spelling is wrong.

Sometimes typos changing the meaning of the words. For example:

Current translation: `сетового API` → `Armor set API` (wrong)
Fixed translation: `сетевого API` → `Network API` (correct)
Current translation: `Амазная бита` → `Accusatory Drill Bit` (wrong)
Fixed translation: `Алмазная бита` → `Diamond Drill Bit` (correct)

Sometimes translation has grammar errors. For example:

- The word  `Puzzle` was translated as  `паззл`. Chosen word is correct but spelling of this word is wrong. It should have only one letter `з` in it. It should be spelled as `пазл`. This is very common mistake.
- The word `Взаймодействие` should be spelled as `Взаимодействие`. This is very common mistake when people mix ups letters `й` and `и` in words.

In other cases we could see incorrect usages of the verbs:

 - Встав(ь)те предметы в слот для запуска.
 - Увелич(ь)те давление до %s бар.
 - Создайте УФ-коробку, обеспеч(ь)те давление, и вставьте пустую печатную плату.
 - Смаж(ь)те зельями ваши пули.

In those examples all the verbs should have the letter 'ь' in it because this is imperative verbs and their basic form has it. Other spelling of these verbs is wrong.
 - Вставь
 - Увеличь
 - Обеспечь
 - Смажь

Some words has very minor typos in it. For example there is one or two missing/extra letters in those words.

- `GPS мркер` should be spelled as `GPS м(а)ркер` because there is no word `мркер`.  
- `Сооздайте` should be spelled as `Создайте`, because there is no word `Сооздайте`. 

Also some phrases in the russian translation was a complete tracing-paper from English or is oversaturated with anglicisms so I changed them a little bit.

I could explain every change I've maid but I'm assure you there is nothing that you could call as personal preferences or dialect based fixes.

desht commented:
Very informative, thanks!  Should be able to get this merged in the next day or two.

Commit:
Fixing more typos and code style

Edarlingen commented:
@desht do I need to fix something before this request could be merged?
